packages/replay-data/src/gitUtil/LocalGitRepo.ts

```diff
@@ -41,6 +41,9 @@ export default class LocalGitRepo {
 
   async init(forceDelete: boolean): Promise<void> {
     if (forceDelete) {
+      if (!this.url) {
```

Comment:
NOTE: `forceDelete` is a legacy option, only needed when we annotate (and want to *re-*annotate) code.